File: class/190719/array.java
public class array {
  public static void main(String[] args){
    String[] Mamamoo = {"화사","휘인","솔라","문별"};
    System.out.println(Mamamoo[3]);
    System.out.println(Mamamoo);
    String[] MamamooC = Mamamoo;
    MamamooC[0] = "청하";
    System.out.println(Mamamoo[0]); // shallow copy
  }
}

File: class/190719/array.js
var Mamamoo = ["화사","휘인","솔라","문별"]
console.log(Mamamoo[3])
console.log(Mamamoo)

MamamooC = Mamamoo
console.log(MamamooC[3])
console.log(MamamooC)
MamamooC[0] = "청하"
console.log(MamamooC[0])
console.log(MamamooC)
console.log(Mamamoo)
// original array is also modified
// due to the shallow copy  
